packages/web/src/shared/languageCodes.ts

```diff
@@ -0,0 +1,7920 @@
+// This list of iso639-3 codes is from SIL.
```

Comment:
I seen something on SIL's site about requiring attribution, so hopefully this covers that.